### ISSUE TITLE
Test commands not running on Windows

### ISSUE THREAD

**Issue by nehaljwani** [conda/conda-build#3243]
This is a side effect of https://github.com/conda/conda-build/pull/3236

A very small reproducer is:
```
(base) C:\Users\Wani\Downloads\scratch>type meta.yaml
package:
   name: yes_sir
   version: 0.0.1

test:
   commands:
     - echo "Hello"
```

For the time being, downgrade conda-build to 3.16.1

**Comment by dhirschfeld** [comment]
@msarahan - commit https://github.com/conda/conda-build/commit/a0de016ccf0e9d223b8da5ad12d71ec80a8fcae7 looks a bit suspicious

In particular, the change from using `test_file` to `shell_file`:

![image](https://user-images.githubusercontent.com/881019/48105004-72d4b000-e281-11e8-956e-0985838131fc.png)


That might well be a red-herring though as I'm not sure I follow the logic and haven't properly tried to debug it.

**Comment by gabm** [comment]
I also stumbled across this and fixed it in #3257
